hbt/config/configs_run2ul.py

```diff
@@ -684,7 +657,8 @@ def add_aliases(
 
     # versions per task family and optionally also dataset and shift
```

Comment:
Ah, sorry, the comment above is actually meant for the version names themselves. The "string or True" is just a temporary thing since we don't have a versioning system yet.

Reply:
I'm not quite sure what you mean. If the comment and the actual code mean two different things, I think we should still update the comment (even if it's temporary) to avoid confusion

Reply:
Yup, I disabled the versioning for now, makes more sense that this `or True` thing.